pkg/controllers/machinesync/machine_sync_controller_test.go

```
@@ -525,6 +525,57 @@ var _ = Describe("With a running MachineSync Reconciler", func() {
+								HaveField("Status", Equal(corev1.ConditionTrue)),
+							))),
+					)
+
+					By("Removing finalizers and deleting the CAPI infra machine out-of-band")
+					Eventually(k.Update(capaMachine, func() {
+						capaMachine.SetFinalizers(nil)
+					})).Should(Succeed())
+					Expect(k8sClient.Delete(ctx, capaMachine)).To(Succeed())
+
+					By("Updating the MAPI machine providerSpec")
+					modifiedMAPIMachineBuilder := machinev1resourcebuilder.Machine().
```

golangci-lint (2.11.4)

[error] missing whitespace above this line (invalid statement above assign)

(wsl_v5)